tests/server/test_server.yml

```diff
@@ -0,0 +1,123 @@
+---
+- name: Test server
+  hosts: ipaserver
+  become: true
+
+  tasks:
+
+  # CLEANUP TEST ITEMS
+
+  - name: Ensure server "{{ 'ipaserver.' + ipaserver_domain }}" without location
+    ipaserver:
```

Comment:
Please, add `ipaadmin_password` to all the tests.

Reply:
Done